.github/workflows/test-autodepends.yaml

```
@@ -0,0 +1,79 @@
+name: Run tests identified via dependency analysis
```

Nitpick, but I think having shorter names here helps the UX in quite a few places. E.g. the "Successful in 8m" isn't truncated in the PR dialog showing the CI checks, etc.

```suggestion
name: Test only changed
```